web/devpi_web/views.py

```diff
@@ -673,11 +678,11 @@ def version_get(context, request):
         make_toxresults_url=functools.partial(
             request.route_url, "toxresults",
             user=context.username, index=context.index,
-            project=context.project, version=context.version),
+            project=context.project, version=version),
         make_toxresult_url=functools.partial(
             request.route_url, "toxresult",
             user=context.username, index=context.index,
-            project=context.project, version=context.version))
+            project=context.project, version=version))
 
```

Comment:
Can you please return version=version here as well? It would help me in handling latest/stable versions in my custom theme.

Reply:
I wrongly assumed it would be available via the request already. I now explicitly added it to the result and added the wrapped context as ``_context`` to the other views.